Frontend/UserTech/src/app/services/rent-bike.service.ts

```diff
@@ -13,4 +13,8 @@ export class RentBikeService {
   rent(bike: Bike): Observable<RentResult>{
     return of({result: 'Ok'});
   }
+
+  return(bikeId: number, stationId: number): Observable<any> {
```

Comment:
Nie jestem przekonany czy return to dobra nazwa na funkcję. Zmieniłbym jednak na coś co nie jest słowem kluczowym js'a.